src/global/styles/typography.scss

```diff
@@ -16,6 +16,11 @@ html {
   color: var(--sbb-color-metal-default);
 }
 
+// Hide components as long as they are instantiated
```

Comment:
I think it would be more like "Hiding components until they are instantiated" (?) if I understand correctly. Great btw 😃👍